app/src/main/res/layout/fragment_register_participant_participant_details.xml

```diff
@@ -93,7 +101,7 @@
                     android:layout_weight="3"
                     android:text="@string/generate_Qr_code"
                     android:textAppearance="@style/ButtonTextStyling"
-                    android:visibility = "visible"/>
+                    android:visibility ="@{viewModel.participantUuid != null ? View.GONE : View.VISIBLE}"/>
```

Comment:
what is the purpose of this line? Shouldn't this button always be visible?

Reply:
Resolved.